llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -138,6 +140,14 @@ def IREELinalgExt_ScatterOp : IREELinalgExt_Op<"scatter",
       return getUpdateSliceRank() == 0;
     }
 
+    SmallVector<int64_t> dimensionMap() {
+      SmallVector<int64_t> ret;
+      for (const APInt& elem : getDimensionMap()) {
+        ret.push_back(elem.getLimitedValue());
+      }
```

Comment:
If it is a `I64ArrayAttr`, you can return `extractFromI64ArrayAttr(getDimensionMap())` directly.

Reply:
Done.